distribution/docs/src/main/resources/_contents/_solr-contents/managing-solr-contents.adoc

```diff
@@ -309,3 +309,68 @@ Here are some helpful suggestions for setting up data backups and recoveries:
 * Run a scheduler program that calls the backup URL on a timed basis.
 * Put indicators in place that can detect when data corruption may have occurred.
 * Testing a backup before recovery is possible. A replicated "staging" Solr server instance can be stood up, and the backup can be copied to that system for testing before moving it to the "production" system.
+
+=== High Availability (Optional)
+
+High Availability in ${branding} is used when a highly available Solr service is needed.
+SolrCloud is a highly available distributed capability of Solr.
+In SolrCloud, if a Solr server fails, another Solr server will be able to provide distributed indexing and search capabilities to the ${branding}.
+SolrCloud enables the ${branding} to have high scale and be fault tolerant.
+The instructions on setting up High Availabilty for ${branding} only include setup in a *NIX environment.
+
+==== Zookeeper
+
+Zookeeper is a distributed hierarchical key-value store, which is used to provide a distributed configuration service, synchronization service, and naming registry for large distributed systems.
```

Comment:
We may want to mention somewhere that Zookeeper is needed because SolrCloud uses it to manage its configuration.